lang/talon/talon.py

```diff
@@ -30,16 +29,20 @@
 mod.list("talon_scopes")
 mod.list("talon_modes")
 
-ctx.matches = r"""
+ctx_talon.matches = r"""
 tag: user.talon
 """
-ctx.lists["user.code_common_function"] = {
+ctx_talon.lists["user.code_common_function"] = {
```

Comment:
We don't want to set this list in Python next to the .talon-list right?

Reply:
I would personally be inclined to remove all default list implementations from python where possible. 

the .talon-list is more specific and will take precedence though